RLBotPack/Kamael/Peter.cfg

```diff
@@ -2,12 +2,12 @@
 # Path to loadout config. Can use relative path from here.
 looks_config = ./peter_appaearance.cfg
 maximum_tick_rate_preference = 120
-#supports_early_start = True
+supports_early_start = False
 # Path to python file. Can use relative path from here.
 python_file = ./Kamael.py
 
 # Name of the bot in-game
-name = St. Peter
+name = St. Peter_dev
```

Comment:
Unintentional name change I suspect

Reply:
oh, nice catch. Yeah. Simply forgot I'd done that.

Reply:
It should be fixed now.